### ISSUE TITLE
Mashing the "Menus: Confirm" Button Causes Plenty of Adverse Effects

### ISSUE THREAD

**Issue by ZaptorZap** [tetrio/issues#372]
### Describe the bug/issue
After goofing around in the menu a bit, I had the genius idea of mashing the enter button for maximum enjoyment. One _quick_ mashing session later, I found a reproducible connection error, perhaps involving entering the Tetra League queue more than once.
### Reproducing
1. Head into the main menu, making sure the "Menus: Confirm" button and the "Menus: Down" is bound.
2. Highlight "MULTIPLAYER" with the "Menus: Down" key
3. Mash "Menus: Confirm" at a high rate
4. See the results
**WARNING:** Be warned that this is a surefire way to lose a match without even seeing the matchup. More details below.
### Screenshot
![image](https://user-images.githubusercontent.com/64891705/84865541-ecd93780-b067-11ea-9eef-16f48b2883c1.png)
(the end result)
### Your setup
 - **OS and version**: Windows 10, 1903
 - **Browser and version**: Firefox, 77.0.1 (64-bit)
### Additional context
It would appear multiple connection errors appear, as closing one would reveal another connection error before refreshing the page. **It would also appear this leads to cancelling a match, and therefore, losing a match.** Before finding this bug, I had played eight out of my ten placement matches. After testing on TETR.IO + Desktop V3, I decided to test on Firefox with TETR.IO + disabled. After testing, I found that my matches were set, and I now see myself in Rank B+ territory.

**Comment by ZaptorZap** [comment]
It would appear this has caused me to enter quick play limbo as well. I wasn't aware of the issue until [GrubbsGaben#2301 brought it to the attention of the TETR.IO community discord.](https://discordapp.com/channels/673303546107658242/673303546564968566/722722962968412331) After logging into tetr.io and attempting to join quick play, to get out, I was met with the message "You are already in this room." 
![image](https://user-images.githubusercontent.com/64891705/84873665-1d729e80-b073-11ea-9e86-b59b3ba02e25.png)
Upon contacting TETR.IO moderator gebruikersnaam, and requesting that they kick me, Zaptor, they went ahead and tried, to no avail. They then escalated to disbanding quickplay, to no avail. It would seem quick play still functions, as once my ghost reaches a 1v1 situation, he instantly loses.
![image](https://user-images.githubusercontent.com/64891705/84874382-084a3f80-b074-11ea-8db3-92afcf49e40c.png)
We then attempted to join a custom room, and I successfully created and played a game. The quick play ghost remained however.
![image](https://user-images.githubusercontent.com/64891705/84875071-00d76600-b075-11ea-8db4-bac4c6861b3d.png)

**Comment by o5k** [comment]
Hotfixed.

**Comment by o5k** [comment]
Quick Play issue was hotfixed, but seems like DCs are still possible

**Comment by ZaptorZap** [comment]
Yep, I got it to happen, with a small difference. This time the "GET READY FOR THE NEXT BATTLE" text showed up before disconnecting me and cancelling the game in the opponents favor. [Video of it in action](https://youtu.be/HCDmYhEZVq0)

**Comment by ZaptorZap** [comment]
I... actually couldn't reproduce the issue this time around. [I even ended up matching against someone and the match went fine.](https://cdn.discordapp.com/attachments/673303546564968566/768692330956849192/TETR.IO_2020-10-22_04-21-53.mp4) If I had to guess why this didn't work now, I'd point towards the recent [Rewrite Of The Life update ](https://tetr.io/about/patchnotes/#chlog_5_1_1) update as that changed quite a few things relating to the matchmaking process if I'm understanding the changes correctly. With that said I'm closing the issue as it's worked itself out it'd seem. (I ended up mashing the enter button for all of thirty minutes lmao)
